### PR TITLE
Improve asset-transfer-basic javascript application workflow + bugfix

### PR DESCRIPTION
**Pull request by sijocherian**
- Adding a sample application workflow to demonstrate the chaincodes, similar to the WIP PR for basic go-application (https://github.com/hyperledger/fabric-samples/pull/211).
- Plan is for app.js to replace query.js & invoke.js, to simplify and document-in-code the app workflow of this sample. I plan to use similar code pattern for asset-transfer-private-data  application-javascript.
  - TODO after feedbacks: remove query.js & invoke.js
- Bugfix: `createAsset` param sequence updated to match the chaincode-go `CreateAsset`.

### CODE REVIEW

File: asset-transfer-basic/chaincode-javascript/lib/assetTransfer.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ class AssetTransfer extends Contract {
     }
 
     // createAsset issues a new asset to the world state with given details.
-    async createAsset(ctx, id, color, size, owner, appraisedValue) {
+    async createAsset(ctx, id, color, owner, size, appraisedValue) {
```
Comment (sijocherian) [inline asset-transfer-basic/chaincode-javascript/lib/assetTransfer.js:67]: `createAsset` param sequence updated to match the chaincode-go `CreateAsset` args

Comment (denali49) [reply asset-transfer-basic/chaincode-javascript/lib/assetTransfer.js:67]: If you switch it back, it should be correct if they merge my pr #227.

Comment (denali49) [reply asset-transfer-basic/chaincode-javascript/lib/assetTransfer.js:67]: To match across all chaincodes and applications regardless of language, the order should be: id, color, size, owner, appraisedValue.

Comment (denali49) [reply asset-transfer-basic/chaincode-javascript/lib/assetTransfer.js:67]: I have also reconciled applications (typescript/javascript) to chaincodes.  See also PR's #228 and #229.

**Comment by sijocherian** [comment]
Looking for feedback, on this code pattern for asset-transfer-basic with  a single file containing a sequence of calls with both submitTransaction & evaluateTransaction (thus removing the files `query.js & invoke.js`). 

That seem to me a better structure in progressive set of tutorials like these new samples.

**Comment by sijocherian** [comment]
move to #241

**Pull request closed without merging**